gapis/atom/group.go

```diff
@@ -38,6 +42,20 @@ type Group struct {
 	Spans Spans  // All sub-groups and sub-ranges of this group.
 }
 
+type SubcommandRange struct {
```

Comment:
Not sure about this name - it's suggesting to me that this is a range of subcommands, when the qualified index suggests it represents a single command that can hold sub-commands. Unfortunately with the merge into the `api` package, I'm not too sure what to suggest as a better name without colliding with something else. Maybe all these data structures should be moved out into a subpackage?